scripts/index.js

```diff
@@ -5,22 +5,15 @@ var introduction = 'Aspiring JavaScript Dev producing high quality responsive we
 
 
 var projects = [];
```

Comment:
Instead of having the global array 'projects', we will from now on we will just track the projects on the Project constructor. It takes some getting use to at first, but in JavaScript since functions are objects, the Project constructor can be thought of as an object itself. We add the property 'all' to the object, and then push all our projects into this array. So wherever you used to push into the var projects array, you will now push into the Projects.all array. If this doesn't make sense come hunt me down and we can talk about it in person :) It definitely takes a shift in thinking at first.